cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/ScheduleTaskRunner.java

```diff
@@ -97,6 +105,12 @@ public void launch(Job job) throws Exception {
     }
     systemArgs.put(ProgramOptionConstants.SCHEDULE_NAME, schedule.getName());
     systemArgs.put(ProgramOptionConstants.EVENT_NOTIFICATIONS, GSON.toJson(job.getNotifications()));
+    SatisfiableTrigger trigger = ((SatisfiableTrigger) job.getSchedule().getTrigger());
+    TriggerInfoContext triggerInfoContext = new TriggerInfoContext(job, store);
```

Comment:
So this is just for getting `TriggerInfo` below? Seems more like a private function in this class.